src/NIfTI.jl

```diff
@@ -107,6 +107,17 @@ function byteswap(hdr::NIfTI1Header)
     hdr
 end
 
+"""
+    NIVolume{T<:Number,N,R} <: AbstractArray{T,N}
+An `N`-dimensional NIfTI volume, with raw data of type 
+`R`. Note that if `raw <: Number`, it will be converted to `Float32`. Additionally, the header is automatically
+updated to be consistent with the raw volume. 
+
+# Members
+- `header`: a `NIfTI1Header`
+- `extensions`: a Vector of `NIfTIExtension`s 
+- `raw`: Raw data of type `R` from the volume
```

Comment:
"from the volume": what does that mean? Is it in fact the whole volume or something else?